insights/parsers/numeric_user_group_name.py

```diff
@@ -0,0 +1,68 @@
+"""
+NumericUserGroupName - command ``/usr/bin/grep -c '^[[:digit:]]' /etc/passwd /etc/group``
+=========================================================================================
+
+This parser reads the output of ``/usr/bin/grep -c '^[[:digit:]]' /etc/passwd /etc/group``
+and detects whether there are any user or group names that begin with a digit.
+
+The grep command returns the number of matches for each file. It matches for user and
+group names that begin with a number. The parser only detects whether the sum is 0 or
+nonzero so as to minimize the amount of collected data.
+
+The boolean answers are available through the attributes ``has_numeric_user_group``,
```

Comment:
`nr_numeric_user` and `nr_numeric_group` are now not boolean variables, we'd better update the documentation correspondingly.

Reply:
Thank you for spotting it.